client/src/pages/Login.js

```diff
@@ -0,0 +1,82 @@
+import React, {useState} from 'react';
+import { bindActionCreators, compose } from 'redux';
+import { connect } from 'react-redux';
+import { styles } from './auth-styles';
+import { Paper, TextField, withStyles } from '@material-ui/core';
+import { Link, Redirect } from 'react-router-dom';
+import { login } from '../actions/userActions';
+
+const Login = ({classes, userStore, login}) => {
+    let [email, postEmail] = useState('');
+    let [pass, postPass] = useState('');
+
+    const handleChangePass = (e) => {
+        postPass(pass = e.target.value);
+    };
+    
+    const handleChangeEmail = (e) => {
+        postEmail(email = e.target.value);
+    };
+
+    const postInfo = () => {
+        login({email, password:pass});
```

Comment:
I feel like we need a space between pass and the colon (again if we used prettier.io, we could be consistent here)

Reply:
fixed (added to linter)